db/models/order.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const {STRING, INTEGER, DOUBLE} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  orderNumber: STRING,
+  status: STRING,
+  qty: INTEGER,
+  price: DOUBLE
+}, {
+  getterMethods: {
+    getOrdNum: () =>
+      `SS-${new Date().toISOString().slice(0, 10)}-${Math.floor(Math.random()*100)+400}`
```

Comment:
When an order is placed we will use the getter method to generate the order number for first item and store the order number in the orderNumber column.  Each additional item will use the "addItem" instance method to reference the order number generated for the first item.